test/prow-cluster-operation/actions/get.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package actions
+
+import (
+	"knative.dev/test-infra/test/prow-cluster-operation/options"
+)
+
+func Get(o *options.RequestWrapper) {
+	o.Prep()
+	o.Request.SkipCreation = true
+	Create(o)
```

Comment:
The logic looks strange... `SkipCreation = true` and then `Create`?

Reply:
yes this is a bit strange, a comment is added here